pydicom/multival.py

```diff
@@ -2,13 +2,17 @@
 """Code for multi-value data elements values,
 or any list of items that must all be the same type.
 """
+from pydicom import compat
 
 try:
     from collections.abc import MutableSequence
 except ImportError:
     from collections import MutableSequence
 
 
+EMPTY_VAL = ''
```

Comment:
This is a leftover I forgot to remove. I have tried to print some special value for empty tags, but that  broke a lot of code.